src/common.js

```diff
@@ -4,11 +4,23 @@
 
 var os = require('os');
 var fs = require('fs');
+var path = require('path');
 var glob = require('glob');
 var shell = require('..');
 
 var DEFAULT_ERROR_CODE = 1;
 var shellMethods = Object.create(shell);
+var PATH = 'PATH';
+
+// windows calls it's path 'Path' usually, but this is not guaranteed.
```

Comment:
Hmmm interesting... I asked a clarifying question on nodejs/node#9157. They should be able to confirm if we should rely on the case-insensitivity or not.